hotline/templatetags/markdown.py

```diff
@@ -0,0 +1,9 @@
+from django import template
+import markdown
+
+register = template.Library()
+
+
+@register.filter
+def md(text):
+    return markdown.markdown(text)
```

Comment:
Does this strip out dangerous HTML? If not, you should consider adding a comment so people know not to use it for untrusted data.